src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -140,7 +142,7 @@ export const WorkspaceForm = ({
   >(
     defaultValues?.permissions && defaultValues.permissions.length > 0
       ? defaultValues.permissions
-      : [{}]
+      : []
```

Comment:
Is this a legacy change?

Reply:
No. This change will remove the default first row for workspace create page.